### PR TITLE
Release version 0.3.0

### PR DESCRIPTION
**Pull request by thomaseizinger**
Given the big feature we added, this is worth a minor release!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,15 +4,25 @@ All notable changes to this project will be documented in this file.
...
-## 0.2.1 - 2019-10-04
+## [Unreleased]
+
+## [0.3.0] - 2019-10-17
+
+### Added
+- Deploy an ERC20 contract on the Ethereum node and give tokens to both parties. The address of the token contract is written to the env file as `ERC20_CONTRACT_ADDRESS`.
+
+### Changed
+- Migrate parts of the code base to async/await :tada:
```
Comment (D4nte) [inline CHANGELOG.md]:
```suggestion
- Migrate parts of the code base to async/await 🎉.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,11 +36,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Move the env file from `./.env` to `~/.create-comit-app/env` to improve usability.
 - Only pull docker images if they are not present locally, this allows offline usage of the `start-env` command.
 
-## 0.1.0 - 2019-09-26
+## [0.1.0] - 2019-09-26
 
 First Release 🎉
 
 [Unreleased]: https://github.com/comit-network/create-comit-app/compare/0.2.1...HEAD
```
Comment (D4nte) [inline CHANGELOG.md]:
```suggestion
[Unreleased]: https://github.com/comit-network/create-comit-app/compare/0.3.0...HEAD
```

File: CHANGELOG.md (hunk 1 above)
Comment (thomaseizinger) [reply CHANGELOG.md]: But I like the text-emoji!
And it renders on GitHub :D

Comment (D4nte) [reply CHANGELOG.md]: It does not render in CLion....

**Pull request merged**